webpack/ForemanInventoryUpload/Components/Dashboard/DashboardActions.js

```diff
@@ -66,16 +71,47 @@ export const setActiveTab = (accountID, tabName) => ({
   },
 });
 
-export const downloadReports = accountID => {
-  const url = inventoryUrl(`${accountID}/uploads/file`);
-  return get({
-    key: INVENTORY_REPORTS_DOWNLOAD,
-    url,
-    toastError: error => error.response.message,
-    handleSuccess: () => {
-      window.location.href = url;
-    },
-  });
+export const downloadReports = accountID => async dispatch => {
+  dispatch({ type: REPORT_DOWNLOAD_REQUEST, payload: { accountID } });
+  try {
+    const filename = `report_for_${accountID}.tar.xz`;
+    const { data } = await Axios.get(
```

Comment:
`webpack/assets/javascripts/react_app/redux/API/API.js` contains a wrap for axios, can you use it?